server/main.lua

```diff
@@ -1,11 +1,4 @@
 ESX                = nil
-PlayersHarvesting  = {}
```

Comment:
Don't remove. Just comment out the lines. For debugging references.

Reply:
This was restored po sa commit #3, dinisable ko nalang po yung harvest and crafting sa client side as suggested po.